components/button.css

```diff
@@ -0,0 +1,71 @@
+.btns{
+    display: flex;
+    gap: 20px;
+    padding: 1rem;
+}
+
+.btn{
+    width: fit-content;
+    padding: .5rem;
+    border: none;
+    border-radius: 10px;
+}
+
+.primary-btn{
+    background-color: #00b594;
+    color: white;
```

Comment:
you can maybe use hexadecimal value for white for maintaining consistency but otherwise overall code looks good to me.

Reply:
Thanks, Varun Will Work on it.